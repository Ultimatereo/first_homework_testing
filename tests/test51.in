5
0 2 5 1 1 4
0 0 1 1 4 1
1 3 4 5 2 0
3 3 4 5 5 0
0 4 3 1 1 2
