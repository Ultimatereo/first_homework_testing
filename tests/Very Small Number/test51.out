-0.297297
-0.0675676
0.932432
-0.725225
0.198198
